bin/sandbox

```diff
@@ -68,10 +66,11 @@ fi
 
 cd ./sandbox
 cat <<RUBY >> Gemfile
-gem 'solidus', github: 'solidusio/solidus', branch: '$BRANCH'
+gem 'solidus', github: 'solidusio/solidus', branch: '$SOLIDUS_BRANCH'
 gem 'rails-i18n'
 gem 'solidus_i18n'
 
+gem "solidus_auth_devise", "~> 2.5" # This extension contains a user decorator, so it must be loaded first
```

Comment:
```suggestion
gem "solidus_auth_devise", "~> 2.5" # This extension contains a user decorator, so it must be loaded first.
```

Reply:
Also, I'm not sure if my understanding of the commit message. Please see below my updated version of the commit message, where I make the references to solidus_auth_devise more explicit. Is it correct?

> Fix bin/sandbox script generator

> Because the solidus_auth_devise extension decorates the User class it needs to
be listed in the Gemfile before the extension. Otherwise, the User class will
not be found.

> A flag will still be given to the Solidus generator so that includes
solidus_auth_devise's seeds. It will skip installing it twice however.